src/components/display/Typography.jsx

```diff
@@ -1,38 +1,32 @@
-import React, {Component} from 'react';
+import React, {useState, useEffect, useRef} from 'react';
 import PropTypes from 'prop-types';
-
 import {Box, Typography as MuiTypography} from '@material-ui/core';
 
 /**
  * Typography component from Material UI
  * https://mui.com/components/typography/
  */
-export default class Typography extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {text: props.text};
-  }
-
-  UNSAFE_componentWillReceiveProps = (nextProps, nextContent) => {
-    // Make sure state remain in sync with received props
-    if (nextProps.text !== this.state.text) this.setState({text: nextProps.text});
-  };
-
-  render() {
-    // Props & state
-    const {id, component, variant} = this.props;
-    let {text} = this.state;
-
-    // Render text
-    return (
-      <Box id={id}>
-        <MuiTypography component={component} variant={variant} gutterBottom>
-          {text}
-        </MuiTypography>
-      </Box>
-    );
-  }
-}
+const Typography = (props) => {
+  const {id, component, variant, text: initialText} = props;
+  const [text, setText] = useState(initialText);
+  const prevPropsRef = useRef(props);
+
+  useEffect(() => {
```

Comment:
♻️ 

Different pattern in refreshing / rendering Dash display components (ref earlier in the `Table`).

Reply:
True, it was overly complex, now simplified to:
`useEffect(() => setText(initialText), [initialText]);`